graphql/database/experiments/features.js

```diff
@@ -427,6 +428,43 @@ const features = {
       },
     ],
   },
+  [V4_SUPPORTING_STATEMENTS]: {
+    defaultValue: 'Control',
+    rules: [
+      /* Begin internal overrides */
+      {
+        condition: {
+          env: {
+            $in: ['local', 'dev'],
+          },
+          [`internalExperimentOverrides.${V4_SUPPORTING_STATEMENTS}`]: {
+            $eq: 'Control',
+            $exists: true,
+          },
+        },
+        force: 'Control',
+      },
+      {
+        condition: {
+          env: {
+            $in: ['local', 'dev'],
+          },
+          [`internalExperimentOverrides.${V4_SUPPORTING_STATEMENTS}`]: {
+            $eq: 'Experiment',
+            $exists: true,
+          },
+        },
+        force: 'Experiment',
+      },
+      /* End internal overrides */
+      // {
+      //   variations: ['Control', 'Experiment'],
+      //   weights: [0.1, 0.9],
```

Comment:
@spicermatthews Heads up, the weights should probably be 50/50.

Reply:
Good catch. Fixed.